extensions/wikia/PortabilityDashboard/PortabilityDashboardModel.class.php

```diff
@@ -5,38 +5,81 @@ class PortabilityDashboardModel {
 	const WIKI_ID_FIELD = 'wiki_id';
 	const CUSTOM_INFOBOXES_FIELD = 'custom_infoboxes';
 	const TYPELESS_FIELD = 'typeless';
+	const WIKIS_LIMIT = 500;
 
 	private $connection;
 
-	const WIKIS_LIMIT = 500;
-
 	public function __construct( $db = null ) {
 		$this->connection = $db;
 	}
 
 	/**
-	 * gets model data for portability dashboard
+	 * For wiki url or domain name return portability data
```

Comment:
Please add what format that's in - given the method name it's not obvious that it's a dict wrapped in an array